ros_ws/src/common/constants.h

```diff
@@ -59,6 +67,10 @@ const std::string TELEOP_NODE = "teleop_control";
  * @brief The node name for the teleop_mux node
  */
 const std::string TELEOP_MUX_NODE = "teleop_mux_node";
+/**
+ * @brief The node name for the motor_controller node
+ */
+const std::string MOTOR_CONTROLLER_NODE = "motor_controller_node";
```

Comment:
That's an interesting point!

I think a good middle ground might be to have a separate constants.h file for each package. We will have to duplicate the topic name constants and node names and such, but each node will be an independent unit.

Thoughts?

@ryanloeffelman and @Cazadorro: What do you guys think?